kbase-extension/static/kbase/js/widgets/kbaseTabs.js

```diff
@@ -24,32 +30,26 @@
             ],
         }
     );
-
-    useful methods would be:
-
-    $('#tabs').kbaseTabs('showTab', 'T1');
-    $('#tabs').kbaseTabs('addTab', tabObject);  //the tabObject defined up above
-
 */
 
-define(['kbwidget', 'bootstrap', 'jquery', 'kbaseDeletePrompt'], (
-    KBWidget,
-    bootstrap,
-    $,
-    kbaseDeletePrompt
-) => {
+define([
+    'kbwidget',
+    'jquery',
+    'kbaseDeletePrompt',
+    'widgets/common/ErrorMessage',
+
+    // For effect
+    'bootstrap',
+], (KBWidget, $, kbaseDeletePrompt, $ErrorMessage) => {
     'use strict';
     return KBWidget({
         name: 'kbaseTabs',
 
-        version: '1.0.0',
-
-        _accessors: ['tabsHeight'],
+        version: '1.1.0',
 
         options: {
-            tabPosition: 'top',
```

Comment:
tabPosition not used anywhere, endnote actually implemented in the widget, and not supported by bootstrap 3 afaik, so removed.